index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">    
+    <title>Portfolio</title>
+</head>
+<body>
+    <nav class="navbar">
+        <div class="logo">
+            <a href="">Alexander</a>
+        </div>
+        <ul class="top-navigation">
+            <li class="navigation-item">
+                <a href="">Portfolio</a>
+            </li>
+            <li class="navigation-item">
+                <a href="">About</a>
+            </li>
+            <li class="navigation-item">
+                <a href="">Contact</a>
+            </li>
+        </ul>
+        <div class="menu">
+            <div class="menu-line"></div>
+            <div class="menu-line"></div>
+            <div class="menu-line"></div>
+        </div>
+    </nav>
```

Comment:
- [ ] For semantic purposes, kindly wrap the toolbar section in a `<header>` tag instead of the `<nav>` tag. This will help improve the SEO of your page